zabbix-xxl.json

```diff
@@ -0,0 +1,123 @@
+{
+    "Zabbix-XXL": {
+        "container_links": {
+            "zabbix": [{
+                "name": "zabbox.db",
+                "source_container": "zabbix-db"
+            }],
+            "zabbix-db": [{
+                "name": "zabbix-db-storage",
+                "source_container": "zabbix-db-storage"
+            }]
+        },
+        "containers": {
+            "zabbix-db-storage": {
+                "image": "busybox",
+                "label": "latest",
+                "launch_order": 1,
+                "opts": [
+		            [
+			            "--restart",
+			            "no"
+		            ]
+		        ],
+                "volumes": {
+                    "/var/lib/mysql": {
+                        "description": "Choose a Share for Zabbix Database. (create a share called zabbix-db for this purpose)",
+                        "label": "Zabbix Database",
+                        "min_size": "1073741824"
+                    }
+                }
+            },
+            "zabbix-db": {
+                "image": "monitoringartist/zabbix-db-mariadb",
+                "launch_order": 2,
+                "volumes": {
+                    "/backups": {
+                        "description": "Choose a Share for Zabbix Backups. (create a share called zabbix-backups for this purpose)",
+                        "label": "Zabbix Backups",
+                        "min_size": "1073741824"
+                    }
+                },
+                "opts": [
+                    [
+                        "-e",
+                        "MARIADB_USER=zabbix"
+                    ],
+                    [
+                        "-e",
+                        "MARIADB_PASS=766a2bb5d24bddae466c572bcabca3ee"
+                    ],
+                    [
+                        "--volumes-from",
+                        "zabbix-db-storage"
+                    ],
+                    [
+                        "-v",
```

Comment:
We do this automatically for all containers. Can you confirm it works without this -v option?